extra/update-language-files/index.js

```diff
@@ -61,15 +62,15 @@ for (const file of files) {
 
     // En first
     for (const key in en) {
-        if (! obj[key]) {
+        if (!obj[key]) {
```

Comment:
It might be worth leaving the space between the `!` and `obj[key]` just to fit with the rest of the codebase.